src/utils.rs

```diff
@@ -44,3 +44,23 @@ pub fn decode_base64(input: &str) -> Result<Vec<u8>, String> {
         base64::decode(input).map_err(|err| format!("invalid base64: {:?}", err))
     }
 }
+
+pub fn decode_decimal(input: &str) -> Result<Vec<u8>, String> {
+    input
+        .split_whitespace()
+        .map(|s| s.parse::<u8>().map_err(|err| format!("invalid decimal input: {:?}", err)))
+        .collect::<Result<Vec<u8>, String>>()
```

Comment:
good.
can you also add a filter method to avoid non-decimal chars?
motivation: I want the ability to hash arrays (like `[1, 2, 3, 4]`) without any text manipulation. just paste `[1, 2, 3, 4]` and hash/encrypt/etc it

Reply:
Ack